src/DependencyInjection/SymfonycastsSassExtension.php

```diff
@@ -27,11 +27,15 @@ public function load(array $configs, ContainerBuilder $container): void
         $configuration = $this->getConfiguration($configs, $container);
         $config = $this->processConfiguration($configuration, $configs);
 
+        if (isset($config['embed_sourcemap'])) {
+            $config['sass_options']['embed_source_map'] = $config['embed_sourcemap'];
+        }
```

Comment:
Does it worth mentioning that it's here for BC?

Reply:
I’ll add some comments.. and update the doc :)